runtime/cere/src/lib.rs

```diff
@@ -1429,7 +1447,13 @@ pub type SignedPayload = generic::SignedPayload<RuntimeCall, SignedExtra>;
 pub type CheckedExtrinsic = generic::CheckedExtrinsic<AccountId, RuntimeCall, SignedExtra>;
 
 /// Runtime migrations
-type Migrations = (migrations::Unreleased,);
+type Migrations = (
+	pallet_nomination_pools::migration::versioned_migrations::V5toV6<Runtime>,
+	pallet_nomination_pools::migration::versioned_migrations::V6ToV7<Runtime>,
+	pallet_staking::migrations::v14::MigrateToV14<Runtime>,
+	pallet_grandpa::migrations::MigrateV4ToV5<Runtime>,
+	migrations::Unreleased,
```

Comment:
Be careful to *NOT DEPLOY* the `migrations::Unreleased` anywhere, as it enables the DAC. I think you should remove the `migrations::Unreleased` from the `Migrations`.

But keep the `migrations::Unreleased` module in the code as we will need it for enabling DAC Testnet and Mainnet. 
As for the Qanet - the DAC is enabled already and `migrations::Unreleased` was already executed.